subscription_integration_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/lionelbarrow/braintree-go/testhelpers"
+	"github.com/stretchr/testify/assert"
```

Comment:
I like testify personally too, but the braintree-go package has no external dependencies which makes it much easier for consumers of the package to use it. Can we drop this and replace it with standard Go tests?

Once dependency management becomes simplified in go with vgo, I think we can move to a test framework.